extensions/pkg/controller/worker/genericactuator/actuator_reconcile.go

```diff
@@ -290,7 +290,7 @@ func deployMachineDeployments(
 					},
 				},
 			}
-
+			log.Info("Deploying machine deployment", "name", machineDeployment.Name, "replicas", machineDeployment.Spec.Replicas)
```

Comment:
```suggestion
			log.Info("Deploying machine deployment", "machineDeploymentName", machineDeployment.Name, "replicas", machineDeployment.Spec.Replicas)
```

(ref https://github.com/gardener/gardener/blob/master/docs/development/logging.md)

Reply:
For consistency, shouldn't the field name be also `machineDeploymentReplicas`  instesd of `replicas` ?

Reply:
Not necessarily, I think this is only a special convention regarding "name" keys. Alternatively, you can do this:

```suggestion
			log.Info("Deploying machine deployment", "machineDeployment", client.ObjectKeyFromObject(machineDeployment), "replicas", machineDeployment.Spec.Replicas)
```